src/pages/frontpage/NvaDescriptionSection.tsx

```diff
@@ -16,8 +19,16 @@ const iconStyle = { height: '2rem', width: '2rem' };
 export const NvaDescriptionSection = () => {
   const { t } = useTranslation();
 
+  const registrationQuery = useRegistrationSearch({
+    params: { aggregation: 'all' },
```

Comment:
siden vi ikke skal bruke treffene til noe kan vi droppe den dataen fra body
```suggestion
    params: { aggregation: 'all', results: 0 },
```